src/components/App.js

```diff
@@ -40,7 +40,7 @@ function App() {
       </div>
 
       <div className="space-y-8 px-8 pb-8">
-        <div className="editor">
+        <div className="editor lg:grid-cols-2 sm:grid-cols-1">
```

Comment:
```suggestion
        <div className="editor grid-cols-1 md:grid-cols-2">
```

The `sm:` prefix isn't needed. That should be the default. And I believe `md` is large enough for 2 columns. What do you think?

Reply:
md screen I think is too small for 2 columns. Look at the image below


![image](https://user-images.githubusercontent.com/5365582/82787657-2a7ee200-9e67-11ea-91f8-f9a9382ffedb.png)

Reply:
I like to believe that most developers have larger monitors than the average consumer:

For me having 1 column on large, wouldn't be a good thing for user experience:

![image](https://user-images.githubusercontent.com/1196524/82788316-85fd9f80-9e68-11ea-8e7b-9086d517d4c6.png)

Having 2 columns on large, and 1 on medium or smaller, would be a good thing though:

![image](https://user-images.githubusercontent.com/1196524/82788344-93b32500-9e68-11ea-9aff-79692a1549f5.png)

![image](https://user-images.githubusercontent.com/1196524/82788465-cf4def00-9e68-11ea-82a4-f02a5f190ced.png)

We can make `grid-cols-1 md:grid-cols-2` the default, and add a toggle later that allows users to switch between these two layouts manually?

Reply:
Yes could be great